src/components/Header/Header.jsx

```diff
@@ -38,8 +39,8 @@ export const Header = () => {
       >
         <Flex align="center" justify="flex-start">
           <LogoWalletIcon width="45" height="45" color={iconsThemeColor} />
-          <Heading as="h1" mr="25px" ml="5px" size="lg">
-            BudgetTracker
+          <Heading as="h1" mr="25px" ml="5px" size="lg" color={headerTextColor}>
+            {i18next.t('header.logo.budgetTracker')}
```

Comment:
IMPORTANT!

```suggestion
            {i18next.t('appName')}
```

And also add changes to i18en.json. It should looks like smth like this: 

`
            ...,
            "appName": "PortMoney",
            ...,
`

We already have suggested name for the app so we can name it like PO wanted :)

Reply:
UPD

We already have app name in i18n.json in 'main' from PR #37, so you can merge main into your branch and use  definition from there ;)